client/src/components/Applications/AppForm/AppFormUtils.test.js

```diff
@@ -151,16 +127,4 @@ describe('AppFormUtils', () => {
     const updatedValues = updateValuesForQueue(appFrontera, values);
     expect(updatedValues.maxMinutes).toEqual(120);
   });
-
-  it('updateValuesForQueue avoids updating runtime if not valid or empty', () => {
```

Comment:
ok, now i remember more about https://github.com/TACC/Core-Portal/pull/770/files#r1127073189 

so we were just not updating to min or max time *IF* the users had entered something out of range for all queues or nonsensical (like an empty string).  Hmm, back-in-the-day, i might have been the only one who liked the behavior. So if we drop it, I'm totally fine.